src/index.js

```diff
@@ -1,8 +1,16 @@
+import os from 'os';
 import Promise from 'pinkie';
 import OS from 'os-family';
 import { createClient } from 'browserstack';
 import { Local as BrowserStackLocal } from 'browserstack-local';
+import uid from 'uid';
 import wait from './utils/wait';
+import Hub from './hub.js';
+
+const START_BROWSER_MAX_ATTEMPT = 3;
+const START_BROWSER_TIMEOUT     = 60 * 1000;
+
+const HUB_PORT = 1000;
```

Comment:
Move this settings to options:

`HUB_PORT` -> to BrowserStackConnector constructor options as `servicePort`

Others to:
`startBrowser (browserSettings, url, { jobName, build } = {}, { maxAttepts, openingTimeout, workingTimeout } = {})`